vendor/bat-native-ads/src/bat/ads/inline_content_ad_info.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "bat/ads/inline_content_ad_info.h"
 
+#include "base/check_op.h"
```

Comment:
only need `base/check.h`

Reply:
Fixed in 205d01f9223f4d8c6501fd0a63b2c24559b5b1c3.